src/types.ts

```diff
@@ -0,0 +1,30 @@
+import StyleDictionary, { Config } from 'style-dictionary'
+
+export enum Brand {
+  PRO = 'pro',
+  JEUNE = 'jeune',
+}
+
+export enum Platform {
+  WEB_CSS = 'web.css',
+  WEB_TS = 'web.ts',
+  MOBILE_TS = 'mobile.ts',
+}
+
+export enum Theme {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+export type BrandsConfig = {
+  [key in Brand]: {
+    platforms: Platform[]
+    themes: Theme[]
+  }
+}
+
+export type ConfigFormatter = (sd: typeof StyleDictionary, brand?: string, theme?: string) => Config
+
+export type PlatformConfigs = {
+  [key: string]: ConfigFormatter
```

Comment:
est-ce que ça poserait des problèmes de tout typer strictement ?

```suggestion
export type ConfigFormatter = (sd: typeof StyleDictionary, brand?: Brand, theme?: Theme) => Config

export type PlatformConfigs = {
  [key: Platform]: ConfigFormatter
```